helm/cluster-operator/templates/deployment.yaml

```
@@ -1,18 +1,18 @@
 apiVersion: extensions/v1beta1
```

```suggestion
apiVersion: apps/v1
```

`extensions/v1beta1` is deprecated in K8s 1.16 so I think this should be fixed.

@mig4 I'm changing this in aws-operator so you might https://github.com/giantswarm/aws-operator/pull/2077 helpful.

---

Makes sense to me, question is if we'd want it in the same PR? I'd lean towards splitting it into a separate one, but I see it's not a big change, so I'm OK either way. I see it's a fix so does it mean it needs to go in now as well?

---

followed up in #811